lib/src/utilities/simcompare.dart

```diff
@@ -144,19 +144,19 @@ abstract class SimCompare {
 
   /// Executes [vectors] against the Icarus Verilog simulator.
   static bool iverilogVector(
+    Module module,
     String generatedVerilog,
     String topModule,
```

Comment:
We can make `topModule` optional and just use `module.definitionName` by default.  I believe there's 1 test where we need to override it, but most of the time we can take the default.

Reply:
Seems like dart do not support optional parameters with named parameter.

https://github.com/dart-lang/language/issues/1076

Reply:
Better than using `''` as the "empty" check, you should make `moduleName` nullable: `String? moduleName`.  Then you can just change it to the `definitionName` if it's null: `topModule = moduleName ?? module.definitionName`.

Reply:
Yup, I was thinking the same things.